Extensions/DialogueTree/dialoguetools.js

```diff
@@ -0,0 +1,301 @@
+/**
+ * @memberof gdjs
+ * @class dialoguetree
+ * @static
+ * @private
+ */
+
+gdjs.dialoguetree = {};
+gdjs.dialoguetree.runner = new bondage.Runner();
+
+/**
+ * Load the Dialogue Tree data of the game.
+ * @param {string} sceneVar The path where to save the screenshot
+ */
+gdjs.dialoguetree.loadFromSceneVar = function(runtimeScene, sceneVar, startDialogueNode) {
+	this.runner = gdjs.dialoguetree.runner; //TODO needs to be initiated globally once, outside of any methods
```

Comment:
Is the TODO still necessary? What is blocking?

Reply:
I don't think its needed any more. The runner that holds all dialogue state (user choices made in the past) is currently global already. Still need to test on multiple scenes and files(when we have them) to be sure.